index.d.ts

```diff
@@ -0,0 +1,20 @@
+import { GraphQLFieldResolver, GraphQLResolveInfo  } from 'graphql';
```

Comment:
@mathieutu Would you please use a linter (such as `tslint`) to lint the code, enforcing a consistant coding style? I found that there are inconsistent spaces and semicolons.